### ISSUE TITLE
Edit customer info

### ISSUE THREAD

**Issue by Keeady** [WITS-Inc/POS#5]
When this button (https://github.com/WITS-Inc/POS/blob/development/app/Resources/views/default/customer.html.twig#L52)  is clicked, fetch customer info and populate into a dialog. All fields are changeable except id. Any changes should be sent back to the server when they hit save or update button. The server side implementation goes here https://github.com/WITS-Inc/POS/blob/development/src/AppBundle/Controller/CustomerController.php#L85

**Comment by NeeharikaU** [comment]
Hi Cameria,

Thanks for the clarification. I have not used Twig before so I am reading
up on that before I start making any changes.

**Comment by Keeady** [comment]
Yeah, take your time. You can reuse this function to launch the dialog once you get data from the server https://github.com/WITS-Inc/POS/blob/development/web/js/Customer.js#L3

**Comment by NeeharikaU** [comment]
I have a couple of doubts. Why are we using Twig here? From what I read
Twig is usually used when there are a lot of HTML pages.
Secondly, are we using MVC architecture by any chance? I am asking because
of the Entity and Controller modules.

**Comment by Keeady** [comment]
Yes we are using MVC and REST. Entity is the model. Each url maps to a single resource. For example: www.example.com/customer/100 will map to customer #100.  We do have a lot of html pages. That's why we are using template so we can reuse some parts in different pages.

**Comment by NeeharikaU** [comment]
Oh okay. Thank you for the information. I will start working on the
customer page.
